includes/feeds/google.php

```diff
@@ -556,7 +557,14 @@ public function make_request($id = '', $time_min = 0, $time_max = 0)
 			}
 
 			// Query events in calendar.
-			$response = $google->events->listEvents($id, $args);
+			$simple_calendar_auth_site_token = get_option('simple_calendar_auth_site_token');
+			$response = '';
+			if (isset($simple_calendar_auth_site_token)) {
+				$Oauth_Ajax = new Oauth_Ajax();
+				$response = $Oauth_Ajax->auth_get_calendarsevents($id, $args);
```

Comment:
You are returning "Error" key from "auth_get_calendarsevents". Is that correct way to handle errors previously?

Reply:
I have updated the code to display the error message. also `if ($response instanceof Google_Service_Calendar_Events) {` in this line, we check if the response has a proper instance of event class then it works otherwise we will display the blank calendar.

Reply:
Please use `throw new Google_Service_Exception($response['Error'], 1);`

Reply:
Updated